### PR TITLE
To merge GFDL FV3 dynamic core new version to EMC fork/branch

### PR DESCRIPTION
**Pull request by XiaqiongZhou-NOAA**
To merge GFDL FV3 dynamic core new version (201912 version) to EMC fork/branch 
"Mytile" is changed to "mygrid" in atmos_model.F90 to be consistent with the change in FV3 dynamic core.

Related pull requests:
atmos_cubed_sphere [#15](https://github.com/NOAA-EMC/GFDL_atmos_cubed_sphere/pull/15#issue-375060544)
ufs-weather-model [#58](https://github.com/ufs-community/ufs-weather-model/pull/58#issue-375077725)
ccpp-physics [#397](https://github.com/NCAR/ccpp-physics/pull/397#issue-375082074)

### CODE REVIEW

**Comment by junwang-noaa** [comment]
Thanks for the pull request, Kate. We currently have a commit queue, I will
let you know the date for your commit soon.

File: .gitmodules [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [submodule "atmos_cubed_sphere"]
 	path = atmos_cubed_sphere
-	url = https://github.com/NOAA-EMC/GFDL_atmos_cubed_sphere
+	url =  https://github.com/NOAA-EMC/GFDL_atmos_cubed_sphere
```
Comment (DusanJovic-NOAA) [inline .gitmodules]: Please remove the space before 'https'

Comment (XiaqiongZhou-NOAA) [reply .gitmodules]: It is updated

File: .gitmodules [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [submodule "atmos_cubed_sphere"]
 	path = atmos_cubed_sphere
-	url = https://github.com/NOAA-EMC/GFDL_atmos_cubed_sphere
+        url = https://github.com/NOAA-EMC/GFDL_atmos_cubed_sphere
```
Comment (DusanJovic-NOAA) [inline .gitmodules]: Now TAB is replaced with spaces. Please try to use same formatting as in develop branch. The idea is that we do not introduce any unnecessary changes, so that people do not see any differences. There shouldn't be any change in this file after we merge the code.

Comment (XiaqiongZhou-NOAA) [reply .gitmodules]: Corrected.

**Review by junwang-noaa — APPROVED** [review]

**Review by climbfuji — APPROVED** [review]

**Review by DusanJovic-NOAA — APPROVED** [review]
